.github/CODEOWNERS

```diff
@@ -32,3 +32,18 @@
 /clang/www/make_cxx_dr_status @Endilll
 
 /lldb/ @JDevlieghere
+
+# SPIR-V in MLIR.
+/mlir/include/mlir/Dialect/SPIRV/ @antiagainst @kuhar
+/mlir/include/mlir/Target/SPIRV/ @antiagainst @kuhar
+/mlir/include/mlir/Conversion/SPIRVTo*/ @antiagainst @kuhar
+/mlir/include/mlir/Conversion/*ToSPIRV/ @antiagainst @kuhar
+/mlir/lib/Dialect/SPIRV/ @antiagainst @kuhar
+/mlir/lib/Target/SPIRV/ @antiagainst @kuhar
+/mlir/lib/Conversion/SPIRVTo*/ @antiagainst @kuhar
+/mlir/lib/Conversion/*ToSPIRV/ @antiagainst @kuhar
+/mlir/test/Dialect/SPIRV/ @antiagainst @kuhar
+/mlir/test/Target/SPIRV/ @antiagainst @kuhar
+/mlir/test/lib/Dialect/SPIRV/ @antiagainst @kuhar
+/mlir/unittests/Dialect/SPIRV/ @antiagainst @kuhar
+/mlir/tools/mlir-tblgen/SPIRVUtilsGen.cpp @antiagainst @kuhar
```

Comment:
Can you use a wildcard? `/mlir/**/*SPIRV* ?

Reply:
I didn't know if codeowners supported recursive wildcards -- the documentation seems quite sparse to me: https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#codeowners-syntax:
> A CODEOWNERS file uses a pattern that follows most of the same rules used in [gitignore](https://git-scm.com/docs/gitignore#_pattern_format) files.

and I'm not sure how to test it best. I will try it tomorrow.

Reply:
In the past I tested it locally by modifying my .gitignore file and checking what git was saying when I modified some paths :)

Reply:
I checked with gitignore and it does work. Thanks! https://github.com/llvm/llvm-project/pull/72485